cms/djangoapps/contentstore/tests/test_i18n.py

```diff
@@ -153,9 +153,11 @@ def test_message_catalog_translations(self):
         with mock.patch('gettext.translation', return_value=_translator(domain='text', localedir=localedir,
                                                                         languages=[get_language()])):
             i18n_service = self.get_module_i18n_service(self.descriptor)
-            self.assertEqual(i18n_service.ugettext('Hello'), 'Hello')
-            self.assertNotEqual(i18n_service.ugettext('Hello'), 'fr-hello-world')
-            self.assertNotEqual(i18n_service.ugettext('Hello'), 'es-hello-world')
+            gettext_variant = 'ugettext' if six.PY2 else 'gettext'
```

Comment:
Use the helper that exists instead: https://github.com/edx/edx-platform/blob/master/openedx/core/lib/edx_six.py#L9

Example of usage: https://github.com/edx/edx-platform/blob/f5f875401aa95d8195969ff5ad4e18cc48c03081/common/lib/capa/capa/inputtypes.py#L464

Reply:
thanks for notifying.I will look into it.